src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1463,6 +1463,22 @@ components:
                 - User
                 - EpochChange
                 - All
+            manifest_accounts_withdrawn_from_filter:
```

Comment:
hmm should we validate these filters somehow? i.e if provided strings are addresses or sthn?

Reply:
Added missing validators